packages/react-components/theme-designer/src/components/AccessibilityChecker/AccessibilityChecker.tsx

```diff
@@ -21,21 +21,23 @@ export const AccessibilityChecker: React.FunctionComponent<AccessibilityCheckerP
   const nonAccessiblePairs: ContrastRatioPair[] = [];
   const accessiblePairs: ContrastRatioPair[] = [];
 
-  const calculateContrastRatio = (foreground: string, background: string, colorPairString: string) => {
-    const bgc: Vec3 = hex_to_sRGB(background);
-    const fgc: Vec3 = hex_to_sRGB(foreground);
+  const calculateContrastRatio = (foreground: string, background: string) => {
+    const theme = (props.theme as unknown) as Record<string, string>;
 
-    const currContrastRatio = getContrastRatio(bgc, fgc);
-    const contrastRatioString = currContrastRatio.toFixed(2);
+    const bghex: string = theme[foreground];
+    const fghex: string = theme[background];
+
```

Comment:
we're pretty confident that bgHex and fgHex will be returned, but lets guard against the case where they don't return a value

if (!bghex || !fghex) return;